extensions/functions_datetime.yaml

```diff
@@ -40,6 +64,7 @@ scalar_functions:
       - args:
           - value: timestamp_tz
           - value: interval_year
+          - value: string
```

Comment:
Where are all these strings coming from (in the other functions as well)? This smells like Arrow timezone shenanigans. Substrait `timestamp_tz` values don't conceptually have timezones, nor do they need them for computations. They represent fixed points in time, typically implemented as offsets to the Unix epoch, which is defined in UTC.

Reply:
If `interval_year` is a physical unit and `timestamp_tz` is in UTC I agree.

Reply:
I've reread and since `interval_year` is a count of days and hence not a physical unit we need the timezone. Imagine the case where you add time to a `timestamp_tz` and cross a DST switch.

Reply:
You're right, I didn't consider that `interval_year` isn't physical. My bad.